packages/xchain-mayachain/package.json

```diff
@@ -33,27 +33,22 @@
     "start:example": "ts-node example/index.ts",
     "generate:MayachainMsgs": "./genMsgs.sh"
   },
+  "dependencies": {
+    "@cosmjs/encoding": "0.31.1",
+    "@cosmjs/crypto": "0.31.1",
+    "@xchainjs/xchain-client": "0.16.1",
+    "@xchainjs/xchain-cosmos-sdk": "0.2.1",
+    "@xchainjs/xchain-util": "0.13.2",
+    "axios": "1.3.6",
+    "bignumber.js": "9.0.0",
+    "protobufjs": "6.11.4"
+  },
   "devDependencies": {
-    "@cosmos-client/core": "0.46.1",
-    "@types/big.js": "^6.1.6",
-    "@xchainjs/xchain-client": "^0.16.1",
-    "@xchainjs/xchain-cosmos": "^0.21.10",
-    "@xchainjs/xchain-crypto": "^0.3.1",
-    "@xchainjs/xchain-util": "^0.13.1",
-    "axios": "^1.3.6",
-    "bech32-buffer": "^0.2.0",
-    "nock": "^13.0.5"
+    "@cosmjs/amino": "0.31.1",
+    "@cosmjs/proto-signing": "0.31.1",
+    "cosmjs-types": "0.8.0"
```

Comment:
Why these dependencies are devDependencies ?

Reply:
While `proto-signing` and `cosmjs-types` should be dependencies, `amino` is devDependency because a type is the only element imported from the package